my.css

```diff
@@ -190,12 +195,12 @@ body {
     position: absolute;
     left: 0;
     top: 0;
-    width: 400px;
+    width: 40%;
     height: 800px;
     background-color: black;
     z-index: -1;
   }
-
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)